server/tests/fixtures/email/emailData.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  validUser: [
+    "Lateefat",
+    "Amuda",
+    "lateefat.amud@andela.com",
+    "wwww.google.com"
+  ],
+  invalidUser1: ["Lateefat", "Amuda", "lateefat.amud", "google.com"],
```

Comment:
Why is this in an array and being designed like an object?

Reply:
I put in an array because I used spread operator to pass it as an argument to the sendVerification parameter. If it is in an object, I will have to dereference each of the key pair properties
I will later have `sendVerification(...emailData.validUser)` instead of 
`sendVerification(emailData.validUser.firstName, emailData.validUser.email, ....) `if I choose to use object

Reply:
what about spreading an object
https://lucybain.com/blog/2018/js-es6-spread-operator/
Now I can use both key-value pair and the spread.